springboot3-todo/Makefile

```diff
@@ -0,0 +1,29 @@
+UNAME := $(shell uname)
+
+ifeq ($(UNAME), Linux)
+	os=linux
+endif
+ifeq ($(UNAME), Darwin)
+	os=mac
+endif
+
+.PHONY: build build-mac build-linux clean
+
+build: build-$(os)
+
+build-mac: target/todo.jar 
+	docker build -q -f src/main/docker/Dockerfile -t spring/todo .
+
+build-linux: target/todo.jar
+	podman build -q -f src/main/docker/Dockerfile -t spring/todo .
+
+target/todo.jar: 
+	./mvnw package
+
+
+clean:
+	./mvnw clean
+
+
```

Comment:
we will want native build targets in here as well, similar to the Quarkus Makefile

Reply:
Are these targets in `spring-todo/Makefile`? I copied the entire `spring-todo` directory as a starting point.